crates/sui-types/src/execution_status.rs

```diff
@@ -234,10 +240,15 @@ pub enum CommandArgumentError {
         Taken values cannot be used again."
     )]
     InvalidValueUsage,
-    #[error("Immutable and shared objects cannot be passed by-value.")]
+    #[error("Immutable objects cannot be passed by-value.")]
     InvalidObjectByValue,
     #[error("Immutable objects cannot be passed by mutable reference, &mut.")]
     InvalidObjectByMutRef,
+    #[error(
+        "Shared object operations such a wrapping, freezing, or converting to owned are not \
```

Comment:
I think this case is only from using the transfer command?

Reply:
I think the more general error message is ok here because it will still be clear what they need to fix and then they understand the full extent of what they can't do with shared objects